src/Operations.jl

```diff
@@ -146,7 +146,7 @@ function collect_project!(ctx::Context, pkg::PackageSpec, path::String, fix_deps
     if haskey(project, "version")
         pkg.version = VersionNumber(project["version"])
     else
-        @warn "project file for $(pkg.name) is missing a `version` entry"
+        # @warn "project file for $(pkg.name) is missing a `version` entry"
```

Comment:
Post-review: how come this is commented out now?

Reply:
Sorry, it was spammy and made people too panicky about Project files.